requirements.yml

```diff
@@ -0,0 +1,4 @@
+---
+collections:
+  - name: ansible.windows
+  - name: community.docker
```

Comment:
Could this make problems?

Reply:
Good question, if its mandatory installation it might..

Reply:
this will enforce the user has to install both dependencies.

On one hand this is good, because in ansible 2.16+ the `docker_container` role is not found (which is only needed for the docker-executor). On the other hand this is not really needed, when you configure a windows-shell runner (but then `ansible.windows` is required for `win_command`)

optional dependencies would fit perfectly. But installing dependencies should never hurt, due to the introduced namespaces.